### PR TITLE
Add Lizzy's text from example branch

### PR DESCRIPTION
**Pull request by etcadinfinitum**
This is an example commit and pull request for ACM's git gud workshop.

### CODE REVIEW

**Review by MayKeziah — APPROVED** [review]
Moo
   mooo
Moo.

**Review by nerdromere — APPROVED** [review]
Yeah looks good!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Git-Gud-Spring2020
 Add your name to a fun message below!
+
+Starfox is my favorite Zoom doggo! --Lizzy
```
Comment (alcatraz3scap3) [inline README.md]:
```suggestion
Starfox is my favorite ZooOOOoOoOooom doggo! --Lizzy
```

**Review by alcatraz3scap3 — APPROVED** [review]
seems like all is well

**Pull request merged**